lib/babel-plugin-package-json-version.js

```diff
@@ -0,0 +1,38 @@
+const { resolve, sep, dirname } = require('path');
+const { existsSync } = require('fs');
+
+const PACKAGE_JSON = 'package.json';
+
+// returns path to closest package.json file while traversing up from given filepath
+// this is compatible with a yarn/lerna workspaces environment
```

Comment:
Did you try this? https://github.com/sindresorhus/read-pkg-up

Reply:
No I haven't, but that looks exactly like what I wanted.  I'll add a commit to use that instead.

Does this project have a preference of `it's-fine-to-include-a-dependency` vs. `this-is-simple-no-need-for-another-dependency`?